src/lib/comms/include/sol-socket.h

```diff
@@ -116,6 +125,19 @@ typedef struct sol_socket_ip_options {
      * sol_socket_bind()
      */
     bool reuse_addr;
+
+    /**
+     * @brief If @c secure is true, this should be considered.
```

Comment:
Why?

There's a `bool secure` inside this same `struct`, and these two fields only apply to the case in which that `bool` is `true`.